inbm/Changelog.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 
+## Fixed
```

Comment:
Since there already is a Fixed Tag for the next release.  You can place it under that one instead of creating a new Fixed tag.